### ISSUE TITLE
Python 3: uniqc broken

### ISSUE THREAD

**Issue by pfuntner** [pfuntner/toys#56]
```
$ echo | uniqc
1 total items
Traceback (most recent call last):
  File "/home/centos/bin/uniqc", line 69, in <module>
    for token in order if args.fifo else sorted(tokens.keys(), cmp=comparator, reverse=True):
TypeError: 'cmp' is an invalid keyword argument for this function
$
```

Apparently the `cmp` argument is no longer expected.  The Python 3 doc points to `functools.cmp_to_key()` but I don't understand what's going on yet or why Python 3 had to make so many freaking changes.  Python 2 rules.

**Comment by pfuntner** [comment]
I haven't solved this yet - I don't think it's an easy fix.  I wanted to merge the branch into master but first I switched the `uniq` script back to Python 2 (2acfe82abc74289d6943a45f389d698a9c486ccf) for now and will keep this issue open.

**Comment by pfuntner** [comment]
Fixed with db7ddac.  It wasn't so hard after all but I don't appreciate that Python 3 had to go and change things.
